### ISSUE TITLE
Where is learning_rate annealing done?

### ISSUE THREAD

**Issue by killerducky** [gcp/leela-zero#399]
I'm looking at tfprocess.py and I see the initial learning_rate, but I don't see where it is annealed. Is that done automatically by tensor flow? 
```
        opt_op = tf.train.MomentumOptimizer(
            learning_rate=0.05, momentum=0.9, use_nesterov=True)

```
I was expecting to see something to implement this from the AGZ doc:
> The learning rate is annealed according to the standard schedule in Extended Data Table 3

**Comment by gcp** [comment]
The schedule is according to the total training steps, but that won't work if you use the net_to_model.py script. It's also not necessarily appropriate for our network configuration/batch size etc, and in the Google paper the training steps are not synchronized to the total amount of learning data, so their table is actually useless unless you can duplicate their setup 100%.

Lowering the learning rate is something you do if the network training stalls out.

**Comment by killerducky** [comment]
Thanks
